server/config/config.json

```diff
@@ -1,8 +1,8 @@
 {
   "development": {
-    "username": "heimdal",
+    "username": "postgres",
     "password": null,
-    "database": "todo_app",
+    "database": "postgres",
```

Comment:
I think creating  a different database would be more appropriate instead of using the default `postgres` database as it has a special use by postgres.

Reply:
Thank you for your feedback. Changes would be made accordingly cheers!